code/socialDistribution/github_activity/github_activity.py

```diff
@@ -0,0 +1,32 @@
+import requests, timeago
+from datetime import timezone, datetime
+from .github_events import *
```

Comment:
Can we avoid importing using the * character to avoid polluting the namespace? It's been creating issues in other parts of the codebase? E.g. `from .github_events import A, B, ...` or `import github_events as events`